packages/runtime/client/rest/src/pagination.ts

```diff
@@ -17,7 +17,7 @@ export type ItemRange = t.TypeOf<typeof ItemRangeGuard>;
 export const itemRanged = (s: string) => unsafeEither(ItemRangeGuard.decode(s));
 
 export interface Page {
-  current: ItemRange;
+  current?: ItemRange;
```

Comment:
in which case we don't have current ? when there is only 1 page ? the semantic is a bit strange

Reply:
when there arent any transactions for example.